### ISSUE TITLE
Tag action support for iam-users and iam-roles

### ISSUE THREAD

**Issue by FireballDWF** [cloud-custodian/cloud-custodian#3160]
Add tagging support to iam-users and iam-roles per https://aws.amazon.com/about-aws/whats-new/2018/11/aws_manage_iam_users_and_roles_by_using_tags/ and https://aws.amazon.com/blogs/security/add-tags-to-manage-your-aws-iam-users-and-roles/

**Comment by kapilt** [comment]
yeah, it looks like we got the tag augmentation for free (unlike ecs which needed manual enable). i was hoping to wait till reinvent to see if resource group tagging api will end up supporting this, before implementing against the iam specific apis, tbd.

**Comment by jtroberts83** [comment]
Sorry for the dup.  Per the docs https://docs.aws.amazon.com/resourcegroupstagging/latest/APIReference/Welcome.html
they still haven't added IAM tagging support for the Resource Tagging API.  Will this proceed then?

**Comment by jtroberts83** [comment]
Is there something that this is waiting on?  We really need IAM tagging for tracking and managing IAM access keys over 90 days. @FireballDWF @kapilt @thisisshi

**Comment by ksarpong** [comment]
We could use it too in our organization. Any help truly appreciated!

**Comment by kapilt** [comment]
so apparently we didn't get tag augmentation for free, but iam user tag augment and actions for tag/remote-tag/mark-for-op with marked-for-op filter started in #3421

**Comment by sri321** [comment]
@kapilt : Is this in the current release , this would greatly help us

**Comment by kapilt** [comment]
Iam user tag actions/filters should be, role needs another pr

**Comment by sri321** [comment]
Just an observation: "actions:umark" is missing

**Comment by kapilt** [comment]
`remove-tag` is unmark, unmark is just an alias, and is deprecated for new tag action support in favor of remove-tag. once we get a deprecation framework, unmark on extant resources is the first thing going in.
